### ISSUE TITLE
jumped to error page when add "INSTALLED_APPS += ["taiga_contrib_ldap_auth"]" to local.py

### ISSUE THREAD

**Issue by jolinx** [ensky/taiga-contrib-ldap-auth#27]
I was able to login and sign up when I installed taiga with the official script.
Then I was asked to add ldap login function, I found this and do it as what they said, but now i jump to error page http://ip/error when I  added `INSTALLED_APPS += ["taiga_contrib_ldap_auth"]` to 
`~/taiga-back/settings/local.py` ,This is my local.py:
`
taiga@localhost:~$ cat  ~/taiga-back/settings/local.py | grep -v ^#
from .common import *

MEDIA_URL = "/media/"
STATIC_URL = "/static/"

SITES["front"]["domain"] = "localhost:8000"

DEBUG = True
PUBLIC_REGISTER_ENABLED = True

DEFAULT_FROM_EMAIL = "no-reply@example.com"
SERVER_EMAIL = DEFAULT_FROM_EMAIL

INSTALLED_APPS += ["taiga_contrib_ldap_auth"]
LDAP_SERVER = 'ldap://192.168.10.51'
LDAP_PORT = 389

LDAP_BIND_DN = 'CN=ldap,DC=tima,DC=local'
LDAP_BIND_PASSWORD = '123321'   # eg.

LDAP_SEARCH_BASE = 'OU=Timauser,DC=tima,DC=local'

LDAP_SEARCH_PROPERTY = 'sAMAccountName'
LDAP_SEARCH_SUFFIX = None # '@example.com'

LDAP_EMAIL_PROPERTY = 'mail'
LDAP_FULL_NAME_PROPERTY = 'name'
`

**Comment by martin-sa** [comment]
With the information provided, I would suggest the same action as I did in [issue 25](https://github.com/ensky/taiga-contrib-ldap-auth/issues/25).

**Comment by cmthomps** [comment]
Same issue here.  I've added "loginFormType": "ldap" to my conf.json file and I'm still seeing the error.  If I comment out the INSTALLED_APPS += ["taiga_contrib_ldap_auth"] line, the site comes up fine.

**Comment by cmthomps** [comment]
Sorry - I figured my problem out.  Digging through some logs I discovered that copying and pasting the config from GitHub into local.py caused some of the lines to indent and python didn't like that.  When I removed all the spaces in front of INSTALLED_APPS += ["taiga_contrib_ldap_auth"] it worked.  I haven't been successful authenticating against my Active Directory server yet but it's not erroring out anymore.
